ui/angular/src/app/job/create-job/batch/batch.component.ts

```diff
@@ -134,8 +134,8 @@ export class BatchComponent implements OnInit, AfterViewChecked {
       return false;
     }
     this.measureid = this.getMeasureId();
-    let time = new Date().getTimezoneOffset() / 60;
-    let timezone = "GMT" + time + ":00";
+    let time = -(new Date().getTimezoneOffset() / 60);
```

Comment:
Looks like a hack (this and next lines). Probably worth to add a comment. Or even extract common function as long as it happens twice in a code.

Reply:
Is there some best practices regarding such logic in angular?

Reply:
IC, this timezone is used for data partition. but can we tell user this critical information in the page?

Reply:
@guoyuepeng @ashutakGG Moved to utility function. Angular is not my specialty, I was really surprised how much complexity Angular community has about it.